.github/workflows/ci_compliance.yml

```diff
@@ -29,3 +33,31 @@ jobs:
           aws-s3-bucket-name: s2n-tls-ci-artifacts
           aws-s3-region: us-west-2
           cdn: https://d3fqnyekunr9xg.cloudfront.net
+
+      # In order to accurately check if there are any uncommitted changes, cleanup any
+      # intermediate artifacts generated during the duvet run.
+      - name: Cleanup intermediate artifacts
+        run: rm -r specs
+        shell: bash
+
+      - name: Extract RFC spec data
+        working-directory: ./compliance
+        run: ./initialize_duvet.sh
+        shell: bash
+
+      - name: Check if there are uncommitted changes
+        run: |
+          # If this fails you need to run `cd compliance && ./compliance/initialize_duvet.sh`
+          #
```

Comment:
If the fix is to run "initialize_duvet.sh", and this diff is produced by running "initialize_duvet.sh", why are we requiring developers to do this manually? If we need to run "initialize_duvet.sh" anyway why not just use the result?

Reply:
It looks like Duvet doesn't do anything if the spec folder already exists: https://github.com/awslabs/duvet/blob/5ed1e4edf75026f83a9fc678a382aa32d16faff7/src/target.rs#L79 So this DOES still save us network calls and make this action less likely to fail.

Reply:
Please put a comment on this so that it's clear why we're doing this, and future developers don't ask the same question :) Without context, this looks very silly.

Reply:
Sadly our duvet reporting is a bit messy at the moment and I found 2 issues with it. This does mean we are downloading the RFC specs in our CI at the moment. I am going to leave a comment linking these issue since ideally we do not re-download the RFC spec each time.

- https://github.com/aws/s2n-tls/issues/4219
- https://github.com/aws/s2n-tls/issues/4220

Reply:
Documented the issue in code as well:
https://github.com/aws/s2n-tls/pull/4205/commits/e988e26dcfe2e8dbc98d59ddabb8cd359a2438a6